docs/testing.md

```diff
@@ -8,10 +8,16 @@ Configure properties, needed to run e2e tests in `e2e/config/application.yml`.
 **Keep in mind that most of the configuration needs to be the same as the application's**.
 It supports the following configurations:
 
-Name                       | Default                              | Description
--------------------------- | ------------------------------------ | ----------------------------------
-TODO: fill the necessary properties.
-
+Name                        | Description
+-------------------------- | ----------------------------------
+`hedera.bridge_account`    | The configured Bridge account. Validators listen for CryptoTranfers crediting that account
+`hedera.topic_id`          | The configured Bridge Topic. Validators watch & submit signatures to that Topic
+`hedera.sender.account`    | The account that will be sending the Hbars through the bridge
+`hedera.sender.private_key`| The private key for the account that will be sending Hbars through the bridge
+`hedera.ethereum.node_url` | The Ethereum Node that will be used for querying data
+`hedera.ethereum.whbar_contract_address` | The ERC20 WHBAR contract on Ethereum
+`hedera.ethereum.bridge_contract_address` | The Bridge contract on Ethereum
+`hedera.validator_url` | The URL of the Validator node. Used for querying Metadata
```

Comment:
Can we format the table in md as well (add whitespaces), just to be readable when editing as well?